data-serving/data-service/src/controllers/case.ts

```diff
@@ -284,13 +283,10 @@ export const batchUpsert = async (
             { ordered: false },
         );
         res.status(207).json({
-            // Types are a little goofy here. We're grabbing the string ID from
-            // what MongoDB returns, which is data in the form of:
-            //   { index0 (string): _id0 (ObjectId), ..., indexN: _idN}
-            createdCaseIds: Object.entries(bulkWriteResult.insertedIds)
-                .concat(Object.entries(bulkWriteResult.upsertedIds))
-                .map((kv) => String(kv[1])),
-            updatedCaseIds: toBeUpsertedCaseIds,
+            numCreated:
+                (bulkWriteResult.insertedCount || 0) +
+                (bulkWriteResult.upsertedCount || 0),
+            numUpdated: res.locals.numModified,
```

Comment:
Why not use `bulkWriteResult.nModified` ? 

"The number of existing documents updated or replaced. If the update/replacement operation results in no change to an existing document, such as setting the value of the field to its current value, nModified can be less than nMatched. Inserted documents do not affect the number of nModified; refer to the nInserted and nUpserted fields instead."
https://docs.mongodb.com/manual/reference/method/BulkWriteResult/

Reply:
I'd anticipated using this, but then noticed that the document's technically modified -- we're still updating the `upload`. I don't think we want a revision for this, though, particularly after we change upload to be an array.